### PR TITLE
Feat/dlcl 776 empty state

### PR DESCRIPTION
**Pull request by SorinOpt**
Related tickets | https://opentechdev.atlassian.net/browse/DLCL-776 

**Thank you for your contribution to the repo. Before submitting this PR, please make sure:**
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests
- [ ] You have updated documentation
- [ ] You have tested your changes

**Please provide a description of the changes proposed in the pull request and reference any related issues in the repository.**

**Please indicate if this PR contains:**
- [ ] A bug fix
- [x] A feature request
- [ ] Breaking changes
- [x] An enhancement

### CODE REVIEW

File: src/demos/assets/agenda.svg
Comment (fadiDL) [inline src/demos/assets/agenda.svg:1]: @SorinOpt where di you get this svg from ? seems like it needs to be in our DlIcons repo

File: tests/DlEmptyState.spec.ts
Comment (fadiDL) [inline tests/DlEmptyState.spec.ts:1]: Need to rewrite this to align with other tests we have in the library.

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: src/demos/assets/agenda.svg
Comment (SorinOpt) [reply src/demos/assets/agenda.svg:1]: how we can use dataloop icons as a background image?

Comment (fadiDL) [reply src/demos/assets/agenda.svg:1]: @SorinOpt  You can acess the svg files from the icons repository
or using the git link itself  see in the dl-icon how we handle loading an svg file from dataloop

**Review by fadiDL — APPROVED** [review]

**Pull request merged**